backend/app/routers/brand.py

```
@@ -111,3 +94,21 @@ async def update_brand(
     db.commit()
     db.refresh(db_brand)
     return db_brand
+
+
+@router.get("/brands/{brand_id}")
+async def get_brand(
+    brand_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)